asf_welsh_energy_consultation/pipeline/plotting.py

```diff
@@ -273,3 +281,15 @@ def age_prop_chart(base_data, title, filename, language="english"):
     plt.savefig(fig_output_path[language] + filename + ".png", bbox_inches="tight")
 
     print("Saved: " + filename + ".png")
+
+
+def plot_kepler_graph(base_data, filename):
+    hex_map = KeplerGl(height=500)
+    hex_map.add_data(
+        data=base_data[["perc_true", "hex_id"]], name="Heat pump proportions"
```

Comment:
one way to go around the name that appears on the plot is to rename "perc_true" before plotting.